packages/components/src/date-time/index.js

```diff
@@ -1,24 +1,29 @@
 /**
  * Internal dependencies
  */
+import 'react-dates/initialize';
+import 'react-dates/lib/css/_datepicker.css';
+
 import { default as DatePicker } from './date';
 import { default as TimePicker } from './time';
 
 export { DatePicker, TimePicker };
 
 export function DateTimePicker( { currentDate, onChange, is12Hour, ...args } ) {
-	return [
-		<DatePicker
-			key="date-picker"
-			currentDate={ currentDate }
-			onChange={ onChange }
-			{ ...args }
-		/>,
-		<TimePicker
-			key="time-picker"
-			currentTime={ currentDate }
-			onChange={ onChange }
-			is12Hour={ is12Hour }
-		/>,
-	];
+	return (
+		<div className="components-datetime">
+			<TimePicker
+				key="time-picker"
+				currentTime={ currentDate }
+				onChange={ onChange }
+				is12Hour={ is12Hour }
+			/>
+			<DatePicker
+				key="date-picker"
+				currentDate={ currentDate }
+				onChange={ onChange }
+				{ ...args }
```

Comment:
Are we using this anywhere? If we aren't maybe it could go for now… if we do want to allow passing props to the child components, we should have a `datePickerProps` we use here and add a `timePickerProps` that is passed to `TimePicker`, for consistency.